scheduler/stores/mongoose/index.js

```diff
@@ -1,9 +1,24 @@
 const mongoose = require('mongoose');
 
 
-function URI({ scheme, host, database }) {
-  return scheme + '://' + host + '/' + database;
-  // return 'mongodb://localhost/test';
+function URI({ scheme, host, port, user, password, database }) {
+  if (process.env.MONGODB_URI) return process.env.MONGODB_URI;
```

Comment:
Maybe we can tackle this in a different way to avoid using process.env but let's keep it this time